pkg/apis/servicecatalog/validation/instance.go

```diff
@@ -60,6 +60,10 @@ func validateInstanceSpec(spec *sc.InstanceSpec, fldPath *field.Path, create boo
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("planName"), spec.PlanName, msg))
 	}
 
+	if spec.Parameters != nil && spec.AlphaSecretParameters != nil {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("alphaSecretParameters"), spec.AlphaSecretParameters, "only one of 'Parameters' and 'AlphaSecretParameters' can be set at a time"))
```

Comment:
@duglin @arschles Secret contains a key-value map here (each will be converted into top-level field in the payload), with every value stored separately I believe. I don't think that multiple secrets will gain anything regarding the size limit there.
This is exactly the same approach as the one used for Bindings already (just in reverse direction), so I wouldn't be concerned about the size limit there. If this approach turns out to have size limitations, it shall be fixed for both Bindings and Instances.